internal/db/schema/migrations/oss/postgres/99/02_worker_authentication.up.sql

```diff
@@ -0,0 +1,147 @@
+begin;
+
+create table worker_auth_operational_indicator_enm(
+  state text primary key
+    constraint only_predefined_worker_auth_states_allowed
+      check (
+          state in ('next','current')
+        )
+);
+
+comment on table worker_auth_operational_indicator_enm is
+  'worker_auth_operational_indicator_enm is an enumeration table representing the state of certificates used for worker authentication.';
+
+insert into worker_auth_operational_indicator_enm (state)
+values
+  ('next'),
+  ('current');
+
+create table worker_auth_ca_certificate(
+ serial_number numeric primary key,
+ certificate  bytea not null
+   constraint certificate_must_not_be_empty
+     check(length(certificate) > 0),
+ not_valid_before  wt_timestamp not null,
+ not_valid_after  wt_timestamp not null
+   constraint not_valid_before_must_be_before_not_valid_after
+     check(not_valid_before < not_valid_after),
+ public_key bytea not null
+   constraint public_key_must_not_be_empty
+     check(length(public_key) > 0),
+ private_key bytea not null -- encrypted PEM encoded private key
+   constraint private_key_must_not_be_empty
+     check(length(private_key) > 0),
+ private_key_hmac bytea not null unique
```

Comment:
What's the purpose of this? The private key is (should be) encrypted with an AEAD so we don't need this for tamper protection, and there's no reason to return this across the API as an identifier so far as I know -- the public key would suffice for that.